ChangeLog

```diff
@@ -1,7 +1,24 @@
-<day of week> <month> <day> 2024 <Author> <author@intel.com>
+<day of week> May <day> 2024 <Author> <author@intel.com>
```

Comment:
why you fill only month?

Reply:
I wanted to make sure I update it with the proper date - anyway I added it already.